cuda10/README.md

```diff
@@ -0,0 +1,76 @@
+# cuda10
+
+This package provides the cuda 10.x libraries and binaries.
+
+[GPU-accelerated Libraries](https://developer.nvidia.com/cuda-zone) for Computing on NVIDIA devices.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+* Ben Dang: <me@bdang.it>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+To compile cuda code (ie. `*.cu`), just add `core/cuda` to your `plan.sh`.  You will then have
+access to compiler tools like `nvcc`, `cuda-gdb` and more.  Furthermore, `LD_RUN_PATH`, `CFLAGS`,
+`CXXFLAGS`, `CPPFLAGS`, and `LDFLAGS` will be updated with any of the cuda shared/static libraries.
+
+If you require shared libraries during runtime, it is recommended that you add `core/cuda-libs` to
+`pkg_deps` and add `core/cuda` to `pkg_build_deps`.
+
+### Example
```

Comment:
to make maintainence easier, you can have a master README.md under `cuda` and just make a note to update the deps so they choose the right major version.